sunpy/map/sources/soho.py

```diff
@@ -114,10 +118,11 @@ def __init__(self, data, header, **kwargs):
         # If non-standard Keyword is present, correct it too, for compatibility.
         if 'date_obs' in self.meta:
             self.meta['date_obs'] = self.meta['date-obs']
-
-        self._name = self.instrument + " " + self.detector + " " + self.measurement
+        self.meta['wavelnth'] = np.nan
+        self.meta['waveunit'] = 'nm'
```

Comment:
This is a bit odd - the wavelength is a NaN in units of nm.  I suppose the reason for definition is the same as given for the STEREO COR maps below.  Is self.meta['wavelnth'] required to be a number?